beam/tools/src/main/java/cz/o2/proxima/beam/tools/groovy/BeamStream.java

```diff
@@ -934,6 +935,15 @@ public <K, S, V, O> Stream<Pair<K, O>> reduceValueStateByKey(
         });
   }
 
+  @Override
+  public Stream<T> reshuffle(@Nullable String name) {
```

Comment:
I don't see the `name` variable used anywhere, is it just for naming the reshuffle operation?

Maybe `unusedName`, or remove if redundant.

Reply:
Beam does not use it for reshuffle, but it might be used in general with some other runner.